File: source/manual/t_b_source/ds_writing/writing_java/StepperMotor4.java
public void write_attr_hardware(Vector attr_list)
{
	Util.out2.println("In write_attr_hardware for "+attr_list.size()+" attribute(s)");

	for (int i = 0;i < attr_list.size();i++)
	{
		int ind = ((Integer)(attr_list.elementAt(i))).intValue();
		WAttribute att = dev_attr.get_w_attr_by_ind(ind);
		String att_name = att.get_name();

		if (att_name.equals("SetPosition") == true)
		{
			attr_SetPosition_write = att.get_lg_write_value();
			Util.out2.println("Attribute SetPosition value = "+attr_SetPosition_write);
			position[0] = attr_SetPosition_write;
		}
	}
}

File: source/development/device-api/device-server-writing/writing/steppermotor4.cpp
void StepperMotor::write_attr_hardware(vector<long> &attr_list)
{
	cout2 << "In write_attr_hardware for " << attr_list.size();
	cout2 << " attribute(s)" << endl;
	
	for (long i = 0;i < attr_list.size();i++)
	{
		Tango::WAttribute &att = dev_attr->get_w_attr_by_ind(attr_list[i]);
		string att_name = att.get_name();

		if (att_name == "SetPosition")
		{
			att.get_write_value(attr_SetPosition_write);
			cout2 << "Attribute SetPosition value = ";
			cout2 << attr_SetPosition_write << endl;
			position[0] = attr_SetPosition_write;
		}
	}
}
